internal/core/data/controller/messaging/subscriber.go

```diff
@@ -48,6 +48,9 @@ func SubscribeEvents(ctx context.Context, dic *di.Container) errors.EdgeX {
 			Messages: messages,
 		},
 	}
+	
+	// Log the topics to which core-data is subscribing
+	lc.Infof("core-data subscribing to topics: %v", topics)
```

Comment:
the log should be put after the subscribe action
please move after line 58
since it's the core-data log, you don't need to add "core-data" into the log
also, it's not clear to print the channel, so you can use for loop ot print the topic only

```
	for _, t := range topics {
		lc.Infof("Subscribed to topics: %s", t.Topic)
	}
```

Reply:
Hello @cloudxxx8, please help to review I have pushed the updated changes. 

Thanks